### PR TITLE
Use sh instead of bash for shell code snippets 

### PR DESCRIPTION
**Pull request by aduh95**
Refs: https://github.com/nodejs/node/pull/34869

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ module.exports.plugins = [
     require("remark-lint-fenced-code-flag"),
     {
       flags: [
-        "bash",
+        "sh",
```
Comment (nschonni) [inline index.js:25]: Looks like we have these alphabetical, can you move it down?

**Comment by nschonni** [comment]
@Trott does this need to be done in 2 phases? EX: one to add the new language, release, then a second one to turn off `bash`?

**Comment by DerekNonGeneric** [comment]
We don't want `sh` on purpose! Look at the reason why. It's confusing.

https://twitter.com/BrendanEich/status/1278803332287483904

**Comment by aduh95** [comment]
@DerekNonGeneric could you elaborate on what is the confusion? The tweet you referenced in the previous comment seems to be about how to name JavaScript/JS, but the discussion here is about replacing `bash` (Bourne Again Shell) with `sh` (Shell Command Language described by the [POSIX standard](http://pubs.opengroup.org/onlinepubs/009695399/utilities/xcu_chap02.html)).

Using `bash` in Node.js docs always seemed weird to me (I'm a `fish` and `zsh` user). Please see [this StackOverflow answer](https://stackoverflow.com/a/5725402/6549112) for more context:

> I prefer `sh` [over `bash`] for the following reasons:
>
> * it is standardized
> * it is portable across POSIX systems — even if they happen not to have `bash`, they are required to have `sh`

**Comment by Trott** [comment]
> @Trott does this need to be done in 2 phases? EX: one to add the new language, release, then a second one to turn off `bash`?

Probably. Another option is to do phase 1 only or at least allow that phase 1 will be very long.

I'm conflicted and ultimately neutral on this change. I'm fine with `bash` being what we use although I agree that `sh` is better semantics. The question is whether those improved semantics are worth all the churn and headache of changing. End users won't see any difference whatsoever. This is entirely for people writing/editing our documentation. Is the hassle of changing worth it? I certainly have a semantic purist streak that wants to see this happen, but pragmatically, I suspect it doesn't ultimately matter. But if someone is willing to do all the work, I won't stop them!

**Comment by aduh95** [comment]
Can this get merge please?

**Comment by DerekNonGeneric** [comment]
> Can this get merge please?

It's not a good idea though, I don't see why you would prefer a more ambiguous info string.

**Comment by aduh95** [comment]
> It's not a good idea though, I don't see why you would prefer a more ambiguous info string.

Are you saying `bash` less ambiguous than `sh`? To me they're just two names, I wouldn't say one is more confusing than the other. I can see you linked to a tweet, but it's a discussion about naming JS language, not a shell, so maybe you meant something else.

I don't want to go too deep into the `bash` bashing (pun intended), but there are good reasons we should use `sh` instead and I think they've already been outlined in this thread and the PR on `node` repo.

**Comment by DerekNonGeneric** [comment]
The point of linking to that tweet was that it's usually better to use the less-confusing (more clear, less likely to be confused) wording. They are also both scripting languages. Would you mind briefly summarizing the case you are making for advocating for `sh`?

**Comment by zackschuster** [comment]
that tweet is about mixing up incompatible languages based on a short-sighted marketing decision from ~25 years ago; `bash` is explicitly, by-specification compatible with `sh`, so it's equivalent to switching from a concrete type to an interface.

**Comment by Trott** [comment]
I'm starting to wonder if it's not worth it to spend energy trying to identify the ideal choice here and we should allow both options. Heck, maybe allow `shell` as well (if it works with our toolchain). It also renders identically (at least in GFM, not sure about highlight.js).

**Comment by aduh95** [comment]
`shell` would be great, but it's unfortunately [an alias to `console` in `highlight.js`](https://github.com/highlightjs/highlight.js/blob/master/SUPPORTED_LANGUAGES.md).

Since @DerekNonGeneric has concerns that hasn't been addressed, I think we should indeed drop it. After all, as @Trott said in an earlier comment, this has no incidence at all on the HTML version of the docs.

**Comment by zackschuster** [comment]
i feel like the "ideal choice" is pretty cut-and-dry — it's part of a specification integral to node itself — and those concerns aren't relevant for reasons stated above. to be frank, i'm confused why there's even a debate. but i do agree this is an exceedingly minor change not worth exhausting oneself over, and if consensus is to drop it then i won't make a fuss :smile:

**Comment by Trott** [comment]
I'm going to close this but feel free to reopen or comment if you feel like more conversation/effort/whatever is warranted.

**Pull request closed without merging**